c2corg_ui/static/js/card.js

```diff
@@ -130,6 +130,34 @@ app.CardController.prototype.translate = function(str) {
 
 
 /**
+ * Show only one of the area types, the first that is available:
+ * 1) range 2) admin limits 3) country
+ * @param {?Array<Object>} areas
+ * @return {string | null}
+ * @export
+ */
+app.CardController.prototype.showArea = function(areas) {
+  if (!app.utils.isTopoguide(window.location.pathname.substring(1)) && areas) {
```

Comment:
For me it's the contrary. The region seems to be an important data in cards. So let's include them,
And then we have to redesign the cards to make the content more optimal. If you make the changes and then add the regions, you will certainly have to redesign everything from scratch.